### DISCUSSION TITLE
Spotify API doesn't work if you're in a spotify private session

### DISCUSSION THREAD

**Discussion by dvdcd** [jwallet/spy-spotify#385] [Q&A] [answered]
I just spent 20 minutes trying to figure out why it wouldn't work, to realize that because I put spotify in private session to keep discord, lastfm, and spotify wrapped from tracking my stats, spytify couldn't grab the now playing data. 
A "no, duh" moment for sure, so i'd like to save future people from my pain lol

**Comment by jwallet** [comment, marked as answer]
self resolved
